src/shared/constants.h

```diff
@@ -94,6 +94,9 @@ static const double SECONDS_PER_MILLISECOND      = 1.0 / 1000.0;
 static const double MILLISECONDS_PER_MICROSECOND = 1.0 / 1000.0;
 static const double MILLISECONDS_PER_NANOSECOND  = 1.0 / 1000000.0;
 
+static const double SIXTY_HERTZ_MILLISECONDS_PER_TICK =
```

Comment:
Can we instead pull this out to a separate file as a configurable constant for the simulator? If thats not possible can we rename this to simulator_tickrate or anything that doesnt have the value in its name

Reply:
done